tests/SimpleThings/Tests/EntityAudit/BaseTest.php

```diff
@@ -120,10 +120,10 @@ protected function getEntityManager()
      */
     protected function getAuditManager()
     {
-        $auditConfig = new AuditConfiguration();
-        $auditConfig->setCurrentUsername('beberlei');
-        $auditConfig->setAuditedEntityClasses($this->auditedEntities);
-        $auditConfig->setGlobalIgnoreColumns(array('ignoreme'));
+        $auditConfig = new AuditConfiguration($this->auditedEntities, array('ignoreme'));
+        $auditConfig->setUsernameCallable(function () {
+            return 'beberlei';
```

Comment:
A new test that sets `uniqid` as a callback would be best here (showing that the username changes over multiple flush calls)